package.json

```diff
@@ -10,126 +10,162 @@
     "url": "https://github.com/keeweb/keeweb"
   },
   "dependencies": {
-    "@babel/core": "^7.14.0",
-    "@babel/plugin-external-helpers": "^7.12.13",
-    "@babel/plugin-proposal-class-properties": "^7.13.0",
-    "@babel/plugin-proposal-optional-chaining": "^7.13.12",
-    "@babel/preset-env": "^7.14.1",
-    "@fortawesome/fontawesome-free": "^5.15.3",
+    "@aetherinox/noxenv": "1.0.0",
+    "@babel/cli": "7.24.1",
+    "@babel/core": "7.24.4",
+    "@babel/eslint-parser": "7.24.1",
+    "@babel/plugin-external-helpers": "7.24.1",
+    "@babel/plugin-transform-class-properties": "7.24.1",
+    "@babel/plugin-transform-logical-assignment-operators": "7.24.1",
+    "@babel/plugin-transform-nullish-coalescing-operator": "7.24.1",
+    "@babel/plugin-transform-object-rest-spread": "7.24.1",
+    "@babel/plugin-transform-optional-catch-binding": "7.24.1",
+    "@babel/plugin-transform-optional-chaining": "7.24.1",
+    "@babel/preset-env": "7.24.4",
+    "@electron/notarize": "1.2.4",
+    "@fortawesome/fontawesome-free": "6.5.2",
     "@keeweb/keeweb-native-messaging-host": "https://github.com/keeweb/keeweb-connect/releases/download/0.3.5/keeweb-native-messaging-host.tgz",
     "@keeweb/keeweb-native-modules": "https://github.com/keeweb/keeweb-native-modules/releases/download/0.11.6/keeweb-native-modules.tgz",
-    "adm-zip": "^0.5.5",
-    "argon2-browser": "1.15.4",
-    "autoprefixer": "^10.2.5",
-    "babel-cli": "^6.26.0",
-    "babel-eslint": "^10.1.0",
-    "babel-loader": "8.2.2",
+    "adm-zip": "0.5.12",
+    "argon2-browser": "1.18.0",
+    "autoprefixer": "10.4.5",
+    "babel-loader": "9.1.3",
     "baron": "3.0.3",
-    "base64-inline-loader": "^1.1.1",
+    "base64-inline-loader": "2.0.1",
     "base64-loader": "1.0.0",
-    "bourbon": "^7.0.0",
-    "chai": "^4.3.4",
-    "cross-env": "^7.0.3",
-    "css-loader": "^5.2.4",
-    "dompurify": "^2.2.8",
-    "electron": "^13.6.9",
-    "electron-builder": "^22.11.1",
-    "electron-evil-feature-patcher": "^1.2.1",
-    "electron-notarize": "^1.0.0",
-    "electron-osx-sign": "^0.5.0",
-    "eslint": "^7.26.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-config-standard": "^16.0.2",
-    "eslint-plugin-babel": "^5.3.1",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.4.0",
-    "eslint-plugin-promise": "5.1.0",
+    "bourbon": "7.3.0",
+    "chai": "5.1.0",
+    "css-loader": "7.1.1",
+    "css-minimizer-webpack-plugin": "^6.0.0",
+    "dompurify": "3.1.0",
+    "electron": "13.6.1",
```

Comment:
Downgrading the patch version here?

Reply:
In regards to which, the only one that is behind still is Electron, because the remote API has been migrated to a separate package, and has re-structured how it works. The other 3 are the latest and will stick there until before release when I'll give them room to do minor updates.

Reply:
Just electron. Looks like we're specifying 13.6.1 now instead of 13.6.9

Reply:
Ooooh, yeah there was a reason for that, I just haven't re-bumped it yet. There was timeout issues related to links, and 13.69 did something wonky with timeouts I had to tweak.

Reply:
Bump before PR merged.